job-dsl-core/src/test/groovy/javaposse/jobdsl/dsl/views/ListViewSpec.groovy

```diff
@@ -310,56 +310,56 @@ class ListViewSpec<T extends ListView> extends Specification {
 
         where:
         filter || children
-        { ->
-            status {
-                matchType(INCLUDE_UNMATCHED)
-                status(Status.UNSTABLE, Status.STABLE, Status.ABORTED, Status.FAILED, Status.DISABLED)
-            }
-        } | [
+                { ->
```

Comment:
Yes, please revert the modifications. Make sure to select the "Only VCS changed text" option in the reformat code dialog (crtl+alt+shift+L).